# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/telemetry/Histograms.json
# Commit: ef90978439c5
# Full Hash: ef90978439c5541f2646af76d1c599a2cb36b140
# Author: Brad Werth <bwerth@mozilla.com>
# Date: 2022-07-21 03:47:38
# Regressor Bug: 1737682
# File Overlap Count: 1
# Description:
#   Bug 1737682 Part 1: Log whether or not we are hitting the detached state. r=mstange
#   
#   There are 10 enum cases that we track internally, one of which isn't
#   emitted as telemetry. These cases are 0-indexed to match the enum values.
#   
# ==============================================================================

diff -r ed4b0f82a124 -r ef90978439c5 toolkit/components/telemetry/Histograms.json
--- a/toolkit/components/telemetry/Histograms.json	Wed Jul 20 19:59:06 2022 +0000
+++ b/toolkit/components/telemetry/Histograms.json	Wed Jul 20 20:00:26 2022 +0000
@@ -16046,6 +16046,17 @@
     "description": "The amount of async paint tasks queued to the paint thread during a layer transaction.",
     "bug_numbers": [1483245]
   },
+  "GFX_MACOS_VIDEO_LOW_POWER": {
+    "record_in_processes": ["main"],
+    "products": ["firefox"],
+    "alert_emails": ["bwerth@mozilla.com"],
+    "expires_in_version": "never",
+    "releaseChannelCollection": "opt-out",
+    "bug_numbers": [1737682],
+    "description": "MacOS video low power state achieved when enqueueing a video frame.",
+    "kind": "categorical",
+    "labels": ["NotVideo", "LowPower", "FailMultipleVideo", "FailWindowed", "FailOverlaid", "FailBacking", "FailMacOSVersion", "FailPref", "FailSurface", "FailEnqueue"]
+  },
   "PERMISSION_REQUEST_ORIGIN_SCHEME": {
     "record_in_processes": ["main"],
     "products": ["firefox", "fennec"],
